server/app/models/ApplicationModel.java

```diff
@@ -75,6 +78,10 @@ public Optional<String> getSubmitterEmail() {
     return Optional.ofNullable(this.submitterEmail);
   }
 
+  public Optional<EligibilityDetermination> getEligibilityDetermination() {
+    return Optional.ofNullable(this.eligibilityDetermination);
```

Comment:
question (maybe blocking): When would this be null? I think with the default value you're setting we're guaranteed to always have an `eligibilityDetermination` and you don't have to worry about handling null, returning an optional from this method, and then handling optional everywhere else in the app. You should just be able to return `EligibilityDetermination` but let me know if I'm missing something.

Reply:
Agreed yeah, it's either default when we create it or default from the database.